Source/NeonAlignable.swift

```diff
@@ -125,69 +125,69 @@ public extension Alignable {
     /// match the specified `Align`. For example, if you specify `.ToTheRightMatchingTop` and provide an offset value of `5`, the
     /// view's y origin will be lower than the sibling view's y origin by 5 points.
     ///
-    public func alignAndFillWidth(align: Align, relativeTo sibling: Frameable, padding: CGFloat, height: CGFloat, offset: CGFloat = 0) {
+    public func alignAndFillWidth(_ align: Align, relativeTo sibling: Frameable, padding: CGFloat, height: CGFloat, offset: CGFloat = 0) {
```

Comment:
It appears as though the rest of the functions that were also changed should probably be changed back. When I went to do the initial Swift 3 updates I added _ to the functions that made sense and left everything else as explicit on purpose.